src/mmwtex.c

```diff
@@ -3047,55 +3057,94 @@ void printTexLongMath(nmbrString *mathString,
     if (!g_oldTexFlag) {
       if (refType == 'e' || refType == 'f') {
         // A hypothesis - don't include \ref{}
-        printLongLine(cat("  ",
-            // If not first step, so print "\\" LaTeX line break
-            !strcmp(htmStep, "1") ? "" : "\\\\ ",
-            htmStep, // Step number
-            " && ",
-            " & ",
-            texLine,
-            // Don't put space to help prevent bad line break
-            "&\\text{Hyp~",
-            // The following puts a hypothesis number such as "2" if
-            // $e label is "abc.2"; if no ".", will be whole label.
-            right(htmRef, instr(1, htmRef, ".") + 1),
-            "}\\notag%",
-            // Add full label as LaTeX comment - note lack of space after
-            // "%" above to prevent bad line break.
-            htmRef, NULL),
-            "    \\notag \\\\ && & \\qquad ", // Continuation line prefix
-            " ");
+        texFull = cat("  ", // texFull[0] should not be a "{" character.
```

Comment:
Here I added a comment to point out that `texFull[0] = " "` should not be a "{" character, since it wouldn't be checked by the subsequent counting algorithm.

Reply:
Ok, and here it obviously will never be "`{`", since it is a blank space "` `".